test.sh

```diff
@@ -325,36 +324,6 @@ souffle_tests7() {
 
 # 'd3log' test group.
 
-tcp_channel() {
```

Comment:
You also want to remove stuff at line 107 (github doesn't let me annotate lines that haven't changed)

Reply:
fixed. thanks